UI for Forgot Password Page/page.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
```

Comment:
The HTML/CSS looks good, but this is a Next project, which uses React, so we need everything to be in component form. In this case, it would be a Page component since it's the ForgotPasswordPage, so please do that. The filepath to this file should be src/app/forgot-password to ensure it routes correctly using the Next.js App Router we went over in Monday's meeting. Make sure to do this for future frontend tasks as well.